src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/X509Certificates/X509AuthorityKeyIdentifierExtension.cs

```diff
@@ -0,0 +1,575 @@
+        /// </param>
+        /// <returns>
+        ///   The configured extension.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="subjectKeyIdentifier"/> is <see langword="null" />.
+        /// </exception>
+        public static X509AuthorityKeyIdentifierExtension CreateFromSubjectKeyIdentifier(
+            X509SubjectKeyIdentifierExtension subjectKeyIdentifier)
+        {
+            ArgumentNullException.ThrowIfNull(subjectKeyIdentifier);
+
+            return CreateFromSubjectKeyIdentifier(
+                subjectKeyIdentifier.SubjectKeyIdentifierBytes.Span);
+        }
+
+        /// <summary>
+        ///   Creates an <see cref="X509AuthorityKeyIdentifierExtension"/> that specifies
+        ///   the provided key identifier value.
+        /// </summary>
+        /// <param name="subjectKeyIdentifier">
+        ///   The subject key identifier value from the Certificate Authority (CA) certificate
+        ///   that will sign this extension.
+        /// </param>
+        /// <returns>
+        ///   The configured extension.
+        /// </returns>
+        /// <exception cref="ArgumentNullException">
+        ///   <paramref name="subjectKeyIdentifier"/> is <see langword="null" />.
+        /// </exception>
+        public static X509AuthorityKeyIdentifierExtension CreateFromSubjectKeyIdentifier(
+        byte[] subjectKeyIdentifier)
+        {
+            ArgumentNullException.ThrowIfNull(subjectKeyIdentifier);
+
+            return CreateFromSubjectKeyIdentifier(new ReadOnlySpan<byte>(subjectKeyIdentifier));
+        }
+
+        /// <summary>
+        ///   Creates an <see cref="X509AuthorityKeyIdentifierExtension"/> that specifies
+        ///   the provided key identifier value.
+        /// </summary>
+        /// <param name="subjectKeyIdentifier">
+        ///   The subject key identifier value from the Certificate Authority (CA) certificate
+        ///   that will sign this extension.
+        /// </param>
+        /// <returns>
+        ///   The configured extension.
+        /// </returns>
+        /// <seealso cref="X509SubjectKeyIdentifierExtension.SubjectKeyIdentifierBytes"/>
+        public static X509AuthorityKeyIdentifierExtension CreateFromSubjectKeyIdentifier(
+            ReadOnlySpan<byte> subjectKeyIdentifier)
+        {
+            AsnWriter writer = new AsnWriter(AsnEncodingRules.DER);
+
+            using (writer.PushSequence())
+            {
+                writer.WriteOctetString(subjectKeyIdentifier, new Asn1Tag(TagClass.ContextSpecific, 0));
+            }
+
+            // Most KeyIdentifier values are computed from SHA-1 (20 bytes), which produces a 24-byte
+            // value for this extension.
+            // Let's go ahead and be really generous before moving to redundant array allocation.
+            Span<byte> stackSpan = stackalloc byte[64];
+            ReadOnlySpan<byte> encoded = stackSpan;
```

Comment:
```suggestion
            scoped ReadOnlySpan<byte> encoded;
```

Reply:
This was marked as resolved, but I don't see the change.  Did you decide not to do it?

Reply:
I hit resolve right before you started reviewing, and haven't pushed yet :smile:

Reply:
Sorry.  I'm usually good about handling race conditions :)